docs/platform/custom-cloud-credentials/troubleshooting.md

```diff
@@ -0,0 +1,30 @@
+# Bring Your Own Cluster (BYOC) Troubleshooting
+
+## Deploying BYOC
+
+### Kubernetes API Authentication Errors
+AWS STS regional endpoints have to be enabled in the target region. Go to https://console.aws.amazon.com/iam/home#/account_settings and verify the regional endpoint is activated. In most cases your region already has AWS STS regional endpoint enabled -> https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html
+
+!!! note
```

Comment:
add this line to the top of the page: `import Note from "@site/src/components/Note";`
and then wrap the note in `<note></note>` instead of using the !!! note syntax